content/en/hosting/4.x/production/google-cloud/_index.md

```diff
@@ -0,0 +1,124 @@
+---
+title: "Production Hosting CHT 4.x - Google Cloud Platform"
+linkTitle: "Google Cloud Platform"
```

Comment:
Is this single or multi-node couch?  Assuming single, this will bette match the other document's link title ("_k3s Multiple Node_" - which actually should be VMWare Multiple Node :shrug: )

```suggestion
linkTitle: "GCP Single Node"
```

Reply:
This is written/scoped for Muso, which is a clustered, and the first request we have for GCP.